.github/stale.yaml

```diff
@@ -0,0 +1,35 @@
+# Settings only applied to pull requests
+pulls:
+  # Number of days of inactivity before an issue becomes stale
+  daysUntilStale: 14
+  # Number of days of inactivity before a stale issue is closed
+  daysUntilClose: 7
+  # Issues with these labels will never be considered stale
+  exemptLabels: []
+  # Label to use when marking an issue as stale
+  staleLabel: stale
+  # Comment to post when marking an issue as stale. Set to `false` to disable
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs. Thank you
+    for your contributions.
+  # Comment to post when closing a stale issue. Set to `false` to disable
+  closeComment: >
+    This pull request has been automatically closed because it has not had recent
+    activity. Thank you for your contributions.
+
+# Settings only applied to issues
+issues:
+  # Number of days of inactivity before an issue becomes stale
+  daysUntilStale: 60
+  # Issues with these labels will never be considered stale
```

Comment:
We don't need `daysUntilClose` because we don't want to close issues automatically, right?

Reply:
Yes.